app/code/Magento/CustomerGraphQl/Model/Resolver/Customer/Account/RevokeCustomerToken.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\CustomerGraphQl\Model\Resolver\Customer\Account;
+
+use Magento\Authorization\Model\UserContextInterface;
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Exception\GraphQlAuthorizationException;
+use Magento\Framework\GraphQl\Query\Resolver\Value;
+use Magento\Framework\GraphQl\Query\Resolver\ValueFactory;
+use Magento\Framework\GraphQl\Query\ResolverInterface;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+use Magento\Integration\Api\CustomerTokenServiceInterface;
+
+/**
+ * Customers Revoke Token resolver, used for GraphQL request processing.
+ */
+class RevokeCustomerToken implements ResolverInterface
+{
+    /**
+     * @var UserContextInterface
+     */
+    private $userContext;
+
+    /**
+     * @var CustomerTokenServiceInterface
+     */
+    private $customerTokenService;
+
+    /**
+     * @var ValueFactory
+     */
+    private $valueFactory;
+
+    /**
+     * @param UserContextInterface $userContext
+     * @param CustomerTokenServiceInterface $customerTokenService
+     * @param ValueFactory $valueFactory
+     */
+    public function __construct(
+        UserContextInterface $userContext,
+        CustomerTokenServiceInterface $customerTokenService,
+        ValueFactory $valueFactory
+    ) {
+        $this->userContext = $userContext;
+        $this->customerTokenService = $customerTokenService;
+        $this->valueFactory = $valueFactory;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function resolve(
+        Field $field,
+        $context,
+        ResolveInfo $info,
+        array $value = null,
+        array $args = null
+    ): Value {
+
+        $customerId = (int) $this->userContext->getUserId();
+
+        if ($customerId === 0) {
+            throw new GraphQlAuthorizationException(
+                __(
+                    'Current customer does not have access to the resource "%1"',
+                    [\Magento\Customer\Model\Customer::ENTITY]
+                )
+            );
+        }
+
+        $result = function () use ($customerId) {
+            return $this->customerTokenService->revokeCustomerAccessToken($customerId);
+        };
+
+        return $this->valueFactory->create($result);
```

Comment:
Pls, remove return `Value`. It's need only for deferred cases. In our case, we can return result of ` $this->customerTokenService->revokeCustomerAccessToken($customerId);`
See updated `ResolverInterface`